packages/gatsby/src/utils/browserslist.ts

```diff
@@ -36,3 +36,8 @@ export const hasES6ModuleSupport = (directory: string): boolean => {
   const browserslist = getBrowsersList(directory)
   return query(browserslist + `, not supports es6-module`).length === 0
 }
+
+export const hasIESupport = (directory: string): boolean => {
+  const browserslist = getBrowsersList(directory)
+  return query(browserslist + ` and ie > 1`).length > 0
```

Comment:
Maybe es6-module is a better fit. es6-module roughly checks of all es6 features like Array.from
```suggestion
  return query(browserslist + ` and supports es6-module`).length > 0
```

Reply:
Good call. That actually simplified this diff quite a bit.